pkg/clients/aws/client.go

```diff
@@ -92,23 +94,38 @@ func (c *client) getAccountNumber(ctx context.Context) (string, error) {
 }
 
 var (
```

Comment:
Nit: it'd be good to make all these constants and bring in the `entitlementUnit` one into the block. Since you need to take pointers of a couple of them in `getLicenseForProductID`, you can use `aws.String()` and `aws.Int32()`.

Reply:
Given the state of this project, I'd prefer to avoid refactoring existing things that work, especially for readability-type concerns like this. I'll keep this in mind though, thanks for the idea on the `aws.String()` and `aws.Int32()`